src/test/java/org/kainos/ea/integration/JobControllerIntegrationTest.java

```diff
@@ -35,18 +32,9 @@ void getAllJobs_ShouldReturnListOfJobs() {
         Assertions.assertNotNull(response.getEntity());
 
         // Check that the all jobs in the response contain the correct properties
-        List<Job> allJobs = response.readEntity(new GenericType<List<Job>>() {});
-
-        for (Job job : allJobs) {
-            int jobID = job.getJobID();
-            Assertions.assertTrue(jobID > 0);
-
-            Assertions.assertNotNull(job.getTitle());
-            Assertions.assertNotNull(job.getJobURL());
-            Assertions.assertNotNull(job.getJobSpec());
-            Assertions.assertNotNull(job.getCapability());
-            Assertions.assertNotNull(job.getJobBand());
-        }
+        List<Job> allJobs = response.readEntity(List.class);
+
+        Assertions.assertFalse(allJobs.isEmpty());
 
     }
```

Comment:
need an integration test for new post endpoint. 1 for successful create and 1 for validation error scenario